testing/wireguard/APKBUILD

```diff
@@ -0,0 +1,94 @@
+# Contributor: Stuart Cardall <developer@it-offshore.co.uk>
+# Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
+
+_flavor=${FLAVOR:-grsec}
```

Comment:
I don't see the point of this if you're hardcoding dependency on `linux-grsec-dev`.

Realistically, we need a better framework for handling kernels, but this seems pointless.